content/en/dashboards/ddsql_editor/reference/tags.md

```diff
@@ -8,9 +8,11 @@ DDSQL is in private beta.
 
 Tags are a widespread mechanism to encode metadata about a particular record across several products at Datadog. Tags are key-value pairs for which a key may contain multiple values.
 
+Tags are modeled in DDSQL with the key as a column name, and values in a `group` type, a sorted set of strings with tag-like "= is contains" semantics.
+
 ## Equality comparisons
 
-Equality comparisons with tags are treated as a "contains" comparison rather than requiring strict equality. `service='website'` is true if a record has a `service` tag with the value `website`, even if it has other service tags as well.
+Equality comparisons between tags and strings are treated as a "contains" comparison rather than requiring strict equality. `service='website'` is true if a record has a `service` tag with the value `website`, even if it has other service tags as well.
```

Comment:
I wanted to clarify that the "contains" behavior applies specifically to equality between tags and strings; as noted in the following section, equality between tags and tags (e.g. `group1 = group2`) is strict equality.